tests/integration_tests/data_steward/utils/participant_summary_requests_test.py

```diff
@@ -0,0 +1,190 @@
+"""
+Integration Test for the deactivated_participants module
+
+Ensures that get_token function fetches the access token properly, get_deactivated_participants
+    fetches all deactivated participants information, and store_participant_data properly stores all
+    the fetched deactivated participant data
+
+Original Issues: DC-797, DC-971 (sub-task), DC-972 (sub-task)
+
+The intent of this module is to check that GCR access token is generated properly, the list of
+    deactivated participants returned contains `participantID`, `suspensionStatus`, and `suspensionTime`,
+    and that the fetched deactivated participants data is stored properly in a BigQuery dataset.
+"""
+
+# Python imports
+import mock
+import os
+
+# Third party imports
+import pandas
+import pandas.testing
+import google.auth.transport.requests as req
+from google.auth import default
+from dateutil import parser
+
+# Project imports
+import utils.participant_summary_requests as psr
+from app_identity import PROJECT_ID
+from utils import auth
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import BaseTest
+
+
+class ParticipantSummaryRequests(BaseTest.BigQueryTestBase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+        super().initialize_class_vars()
+
+        cls.project_id = os.environ.get(PROJECT_ID)
+        cls.dataset_id = os.environ.get('COMBINED_DATASET_ID')
+
+        cls.tablename = '_deactivated_participants'
+        cls.destination_table = cls.dataset_id + '.' + cls.tablename
+
+        cls.fq_table_names = [
+            f"{cls.project_id}.{cls.dataset_id}.{cls.tablename}"
+        ]
+        super().setUpClass()
+
+    def setUp(self):
+        self.columns = ['participantId', 'suspensionStatus', 'suspensionTime']
+        self.bq_columns = ['person_id'] + self.columns[1:]
```

Comment:
This would need to be updated if you choose underscore delimiters in favor of camel casing.

Reply:
Updated 👍🏾